README.md

```diff
@@ -41,7 +44,7 @@ jobs:
           echo ::set-output name=current_version::${GITHUB_REF#refs/tags/v}
         shell: bash
       - name: Checkout code
-        uses: actions/checkout@v2
```

Comment:
Why is this marked as `v2` just like it is `v2` on master?

Reply:
The [actions/checkout](https://github.com/actions/checkout) action is on v2 which is the recommended version to use. I just updated the README to reflect that

Reply:
Ok so there is a mistake here, `action/checkout` was marked as `v2` in the example on master but the change in this PR is to mark it as v1 (see https://github.com/mindsers/changelog-reader-action/pull/17/commits/f52f994de7fbec7b677c972343ccd5531cd1b89a#diff-04c6e90faac2675aa89e2176d2eec7d8R47)